main/Main.hs

```diff
@@ -175,6 +176,9 @@ main = do
         . setServerName (toS $ "postgrest/" <> prettyVersion) $
         defaultSettings
 
+  -- the first value in the db-schema configuration parameter is used as default
+  let defaultSchema = head $ configSchemas conf
```

Comment:
I've noted that taking the head of the schemas would make the procs only work for a single schema(not the whole list). This is because we obtain some info about procs in DbStructure.
https://github.com/PostgREST/postgrest/blob/b75e7cef906b1703eb14ffcc32723eacff30ee11/src/PostgREST/DbStructure.hs#L47-L55

We would need to pass the whole list here and change some of the DbStructure queries. I can help with this.